modules/services/cloud-logs/variables.tf

```diff
@@ -3,6 +3,11 @@ variable "bucket_arn" {
   type        = string
 }
 
+variable "account_id" {
```

Comment:
Will this require changes in onboarding-api installActions snippets?

Reply:
Yes, this has already taken care of in the other PR we discussed during the last meeting. Resolving this :)